tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1945,6 +1946,14 @@ def random_hue(image, max_delta, seed=None):
 
   `max_delta` must be in the interval `[0, 0.5]`.
 
+  Usage Example:
+  ```python
+  >>> x = tf.constant([[[1.0, 2.0, 3.0]]])
+  >>> y = tf.image.random_hue(x, max_delta=0.1)
```

Comment:
Can you seed the op? Otherwise each time we run it we'll get a completely different random number, and the doctest will stay broken.

Reply:
Sorry, what does seeding the op mean?

Reply:
Why was this resolved? There's still no seed argument to random_hue and no doctest.

Reply:
Very sorry, I've used the seed argument in random_hue but it still gives a random number like
`array([[[1.125246, 1.      , 3.      ]]], dtype=float32)` and `array([[[1.       , 2.8408904, 3.       ]]], dtype=float32)`. Is typing down the result in the usage example necessary?

Reply:
Very sorry, I've used the seed argument now. How do you make the doctests not expect the exact value on the first digit after the decimal point?

Reply:
is it possible just to do `2....`?

Reply:
here's my code atm
```python
x = tf.constant([[[2.0, 3.0, 2.0]]])
y = tf.image.random_hue(x, max_delta=0.1, seed=1)
print(y.numpy())
[[[2.        3.        2.1610386]]]
```

Reply:
@alextp gentle ping for review